app/ui/browser-modern/views/browser/page/cert-error.jsx

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+import React, { PropTypes, Component } from 'react';
+
+import Style from '../../../../shared/style';
+
+const CONTAINER_STYLE = Style.registerStyle({
+  width: '700px',
+  margin: '0 auto',
+  display: 'block',
+});
+
+const CERTIFICATE_STYLE = Style.registerStyle({
+  backgroundColor: '#ddd',
```

Comment:
Alright, but I disagree with all these theme variables that are for some ugly, unmocked up, barely styled page as a placeholder for a blank page that seems like something broke